src/presentational/mgc-title.ts

```diff
@@ -0,0 +1,73 @@
+import { LitElement, html, css, TemplateResult, nothing } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+
+import '@internetarchive/icon-donate/icon-donate.js';
+
+@customElement('iaux-mgc-title')
+export class MonthlyGivingCircle extends LitElement {
+  @property({ type: String }) titleStyle: 'heart' | 'default' = 'heart';
```

Comment:
Q, non blocking: Do we need this to be customizable? It looks like the only use of it is the heart.

Reply:
will be reusing this component shortly, where we won't use the heart

Reply:
Ah okay.. then maybe suggestion (non-blocking) to rename the `get heart()` method below to something else since it may not always be a heart

Reply:
thanks for catching that nuance. i will update in the next pr, when i edit usage 👍